app/components/modals/NewDatasetModal.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react'
+
+import { NewDatasetModal } from '../../models/modals'
+
+import { connectComponentToProps } from '../../utils/connectComponentToProps'
+
+import { dismissModal } from '../../actions/ui'
+
+import { selectModal, selectSessionUsername } from '../../selections'
+
+import nameFromTitle from '../../utils/nameFromTitle'
```

Comment:
Please keep imports in three blank-line-delimeted groups:
```
[external packages]

[non-components]

[components]

...
```
```suggestion
import { NewDatasetModal } from '../../models/modals'
import { connectComponentToProps } from '../../utils/connectComponentToProps'
import { dismissModal } from '../../actions/ui'
import { selectModal, selectSessionUsername } from '../../selections'
import nameFromTitle from '../../utils/nameFromTitle'
```

Reply:
👍